pkg/plugins/helm/kubernetes_handler.go

```diff
@@ -99,17 +99,23 @@ func (p *KubeHandler) CheckApiVersionsSupported(apiVersions []string) error {
 		return nil
 	}
 
-	client, _, err := p.initKubeClient()
+	_, config, err := p.initKubeClient()
```

Comment:
There are some other scripts need to fix, such as init-helm.sh

Reply:
ok

Reply:
I see there are helm version in .travis.yml and helm_test.go, fixed them together pls.